pystac_client/cli.py

```diff
@@ -24,12 +25,12 @@ def search(url=STAC_URL, matched=False, save=None, headers=None, **kwargs):
             matched = search.matched()
             print('%s items matched' % matched)
         else:
-            items = ItemCollection(search.items())
+            feature_collection = search.get_all_items_as_dict()
             if save:
                 with open(save, 'w') as f:
-                    f.write(json.dumps(items.to_dict()))
+                    f.write(json.dumps(feature_collection))
             else:
-                print(json.dumps(items.to_dict()))
+                print(json.dumps(feature_collection))
```

Comment:
Do we maybe want to use the `StacIO.save_json` here. It's maybe an extreme edge case, but I could see someone wanting to use `orjson` for serialization for performance. This would allow them to:

```
pip install pystac[orjson]
pip install pystac-client
```

and trust that the serialization will happen with `orjson`.

Like I said, kind of an edge case, so if it ends up being a lot of effort it might not be worth it right now.

Reply:
It is just a CLI so write performance isn't super important. Still, doesn't hurt.  What is needed for printing to stdout though isn't `StacIO.save_json` but rather `StacIO._json_dumps`.  Making that change

Reply:
That's true, probably not a big deal.